CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 8.6.3 (not yet released)
+
+### Expaned RegistryAuthSupplier support
+Add RegistryAuthSuppliers for:
+- reading from docker config file (ConfigFileRegistryAuthSupplier)
+- combining multiple suppliers (MultiRegistryAuthSupplier)
+
+Also change the default behavior of DefaultDockerClient.Builder to use the
```

Comment:
I'm not really sure, I'm not good on knowing when to change this. What do you think?